ci-operator/config/openshift/release/openshift-release-master__nightly-4.11-upgrade-from-stable-4.9.yaml

```diff
@@ -2,17 +2,18 @@ releases:
   initial:
     release:
       channel: stable
-      version: "4.8"
+      version: "4.9"
   intermediate:
-    candidate:
+    prerelease:
```

Comment:
you should be able to stick to `cnandidate` here, because we already have `candidate-4.10` with some FCs in it.

Reply:
`candidate` means a release-controller release (nightly or ci), `prerelease` targets constraints from https://amd64.ocp.releases.ci.openshift.org/#4-stable so `prerelease` is closer to stable. 

But you are right, previous version used a latest nightly as an intermediate, let's be consistent.

Reply:
```console
$ grep -A4 intermediate: ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9-from-stable-4.8.yaml 
  intermediate:
    release:
      channel: stable
      version: "4.9"
  latest:
```

So once we get a `stable-4.10`, we'll want a similar stanza for this 4.9->4.10->4.11 job, right?  I think:

```yaml
  intermediate:
    release:
      channel: candidate
      version: "4.10"
```

would be a fair choice here, so we only have to touch that `channel` line when 4.10 GAs, but :shrug: touching a few more lines when 4.10 GAs won't be a big deal either.